### ISSUE TITLE
Make a temporary Github Pages website

### ISSUE THREAD

**Issue by EdoPlantinga** [demodam/demodam.org#22]
To better show the contributing.md, governance.md etc docs. Does not need to be fancy, just to get us started.

**Comment by marcvanandel** [comment]
By default GitHub pages uses the Jekyll engine to generate the static page. I've experience with [MkDocs](https://www.mkdocs.org/) which - I think - is a little richer in features and tuning. And is a little easier to develop with locally.

What is actually needed / expected here? I can set up an automated pipeline which generates a fresh deployment onto a separate branch after each merge into the `main` branch? (I do need admin rights on this repo for that ... )

**Comment by felixfaassen** [comment]
@marcvanandel @EdoPlantinga . @rubenvdlinde reached out to me to setup of a site like openzaak.org. Which is based on react and Gatsby. I told him I'm willing to help but I also would like someone else committed to writing content and working on the website. I'm also actively looking to nicer hybrid solutions where we would use a static site hosted on GitHub where you can both edit directly by committing to Github OR use a built in client side CMS editor for people who are not familier. I already played with some solutions. For now I would suggest: Fork OpenZaak, tweak and add content and then when we are moving looking for a solution which is even more user friendly so people who don't know Git can still contribute to the website.

**Comment by marcvanandel** [comment]
I agree on that @felixfaassen ! GitHub is great for ... well, developers. But especially sites like this need easy accessible and editable content. On the other hand when it's just a landing page how much does that change?

Would you say Gatsby as framework and an npm project would be as easy as mkdocs? (you might to check out my example at [SensRNet](https://github.com/kadaster-labs/sensrnet-home/)?)

**Comment by EdoPlantinga** [comment]
Thanks all! For maintainability / user friendliness I am not to worried, probably there's only a few people adding to it and like Marc said: it will probably not change too much anyway. Most people that update it will be familiar with Github and markdown I suppose. I'll put a DoD to the issue, to be a little more specific. If things get more complicated than that, we can always switch to a CMS based solution, but I kind of like the fact that people can do pull requests on text on this particular website (makes it feel more like a community thing).

**Comment by EdoPlantinga** [comment]
@AlbaRoza can you review the requirements above? Feel free to add / change them.

**Comment by felixfaassen** [comment]
@EdoPlantinga I will make a branch tomorrow with a fork of openzaak.org where we can make a first version. We can than add content together. OpenZaak.org is based on signalen.org ;-)

**Comment by marcvanandel** [comment]
Start Orgs:
- Dimpac
- Conduction
- MakenMedia
- Foundation for Public Code
- Kadaster

**Comment by felixfaassen** [comment]
Thanks @marcvanandel
